packages/manager/apps/dedicated/client/app/billing/autoRenew/agreements/accept-all/accept-all.html

```diff
@@ -0,0 +1,52 @@
+<oui-modal
+    data-primary-action=":: $ctrl.acceptAndNext()"
+    data-primary-label="{{ (
+        ($ctrl.currentAgreementIndex === ($ctrl.agreements.length - 1))
+            ? 'user_agreements_accept'
+            : 'user_agreements_accept_all_accept_and_next'
+    ) | translate }}"
+    data-primary-disabled="!$ctrl.currentAgreement.accepted"
+    data-secondary-action=":: $ctrl.goBack(false, 'success', true)"
+    data-secondary-label="{{:: 'wizard_cancel' | translate }}"
+    data-loading="$ctrl.AgreementUnderProcess"
+    data-on-dismiss=":: $ctrl.goBack(false, 'success', true)"
+>
+    <h6
+        data-ng-bind="'user_agreements_accept_all_count' | translate: {
+        current: $ctrl.currentAgreementIndex + 1,
+        total: $ctrl.agreements.length
+    }"
+    ></h6>
+    <h3 data-ng-bind=":: $ctrl.currentAgreement.name"></h3>
+    <oui-textarea
+        data-name="agreement"
+        data-model="$ctrl.currentAgreement.text"
+        data-rows="10"
+        data-readonly
+    ></oui-textarea>
+    <div class="oui-checkbox mt-3">
+        <input
```

Comment:
Can't a `oui-checkbox` be used here ?

Reply:
The description has a link, which requires `ng-bind-html`. Hence we will need to have the complete html code instead of the component